packager/react-packager/src/DependencyResolver/haste/index.js

```diff
@@ -157,10 +168,10 @@ HasteDependencyResolver.prototype.wrapModule = function(module, code) {
 
   return DEFINE_MODULE_CODE.replace(DEFINE_MODULE_REPLACE_RE, function(key) {
     return {
-      '_moduleName_': module.id,
+      '_moduleName_': convertPath(module.id),
       '_code_': code.replace(replacePatterns.IMPORT_RE, relativizeCode)
                     .replace(replacePatterns.REQUIRE_RE, relativizeCode),
-      '_deps_': JSON.stringify(resolvedDepsArr),
+      '_deps_': JSON.stringify(resolvedDepsArr.map(convertPath))
```

Comment:
I'm not convinced that we need to change runtime code to support windows. Does node, browserify, or windows change the generated output for windows?

If we fixed things internally we can generate the same output for windows or unix, i.e. the bundle could continue to refer to modules using `/`